.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,33 @@
 
 ## BREAKING CHANGE - remove this line if your code doesn't introduce a breaking change
 
-## Description
+## Release Notes Description (public)
 
-<!--- Describe what this feature does
-  REQUIRED
-  USED IN RELEASE NOTES
+<!--- DESCRIPTION USED IN THE RELEASE NOTES
+
+Remove this comment and replace it with the description of your changes for an external audience. 
+This description will be pulled into public release notes.
+
+1. What changes are you introducing? Be clear and provide a detailed overview of your changes.
+2. Why are you introducing the changes? Make the intent of the changes clear.
+   - What do those changes mean for our end users?
+   - Why should they care? 
+   - What is the context of your changes?
+
+Additionally:
+
+- If your changes are breaking changes, provide information on how the change affects our customers and what is required from them to be updated.
+  If you think a migration guide would be useful, let TWs know in advance.
+- If your changes deprecate an API, provide what is the alternative for our customers (if it exists). State clearly that an API became deprecated.
 -->
 
+<!-- [Mark with '[X]' (no spaces) if Release Notes description is not required for this PR. Use only if this PR has no impact for external audience.] -->
+- [ ] Skip Release Notes for this PR
```

Comment:
Sync script takes whole body of description section, so have you removed this from content, or are You sure that this is correct place?